spec/services/doc_auth_router_spec.rb

```diff
@@ -77,7 +77,9 @@
         ),
       )
 
-      response = I18n.with_locale(:es) { proxy.get_results(instance_id: 'abcdef') }
```

Comment:
The name of this test case could be updated to remove reference to "FriendlyError".

Reply:
Gah, thanks! I forgot to check for instances of the UpperCamelCase of the name. Missed just that one. :-(